datalad/support/tests/test_sshconnector.py

```diff
@@ -74,3 +74,39 @@ def test_ssh_manager_close():
 
     ok_(not exists(opj(manager.socket_dir, 'localhost')))
     ok_(not exists(opj(manager.socket_dir, 'datalad-test')))
+
+
+@skip_ssh
+@with_tempfile(mkdir=True)
+@with_tempfile(content="one")
+@with_tempfile(content="two")
+def test_ssh_copy(sourcedir, sourcefile1, sourcefile2):
+
+    remote_url = 'ssh://localhost'
+    manager = SSHManager()
+    ssh = manager.get_connection(remote_url)
+    ssh.open()
```

Comment:
I'm not clear on this point. Did you want `__enter__` to be used so that we don't have to open `SSHManager()` everytime we want to run `ssh.copy()`